### PR TITLE
Support interp kind

### PR DESCRIPTION
**Pull request by zxdawn**
Support setting interpolation kind of coord.

### CODE REVIEW

File: tobac/themes/tobac_v1/feature_detection.py [hunk 1]
```diff
@@ -63,6 +64,13 @@ def feature_detection_multithreshold(
         Flag choosing method used for the position of the tracked
         feature. Default is 'center'.
 
+    coord_interp_kind : str, optional
+        The kind of interpolation as a string. Default is 'linear'.
```
Comment (freemansw1) [inline tobac/themes/tobac_v1/feature_detection.py:68]: I think this should be clearer as to what the interpolation is touching- just add something like "for coordinates" in here.

**Review by freemansw1 — CHANGES_REQUESTED** [review]
Minor change on clarity, otherwise I'm happy with this!

**Comment by zxdawn** [comment]
@JuliaKukulies Yes, please feel free to merge it ;)

**Comment by JuliaKukulies** [comment]
Or actually, I will leave it to @freemansw1 who still needs to approve your changes :) but I guess they will!

**Pull request merged**